### PR TITLE
feat: Addition of FOSSASIA Policies Pages

### LINKED ISSUE #3051: Links to privacy policy and terms page is not working
**Describe the bug**
Please add a privacy policy and terms&conditons pages. Currently, the URL: https://eventyay.com/privacy-policy/ and https://eventyay.com/terms/ is redirecting to a Error 404 page

**Additional context**
Based on the mobile meeting discussion yesterday: https://docs.google.com/document/d/169tq2sr7lF_RnWnXA7hGVEIhQsrHT9tQjB32hHdlUL8/edit

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #3051 

## Additional Details :- 
- Used `Static Pages` as the point was discussed in `App Weekly Meeting` so took it as priority .

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by kushthedude** [comment]
@shreyanshdwivedi @uds5501 Please check the following .

**Comment by kushthedude** [comment]
@niranjan94 @CosmicCoder96 Please Review

File: app/router.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ const router = Router.extend(RouterScroll, {
 });
 
 router.map(function() {
+  this.route('imprint');
+  this.route('terms-condition');
+  this.route('privacy-policy');
```
Comment (uds5501) [inline app/router.js]: Why is this route added when there is no page for privacy policy *yet*

Comment (kushthedude) [reply app/router.js]: We need to add `Privacy Policy` but the server went down , So pushed the two pages to check if it is okay

**Comment by CosmicCoder96** [comment]
@kushthedude Go to admin->content->pages :) These static pages can be created at will.

**Pull request closed without merging**